src/Servant/Reflex/Response.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE ConstraintKinds   #-}
+{-# LANGUAGE FlexibleContexts  #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Response handlers
+--   Usefull for 'clientWithOptsAndResultHandler'
+module Servant.Reflex.Response
```

Comment:
Mind renaming the module to `Servant.Reflex.ReloadOnError`, since for now that's the scope of the module?